beacon-chain/core/electra/consolidations.go

```diff
@@ -207,7 +207,7 @@ func ProcessConsolidationRequests(ctx context.Context, st state.BeaconState, req
 
 		if npc, err := st.NumPendingConsolidations(); err != nil {
 			return fmt.Errorf("failed to fetch number of pending consolidations: %w", err) // This should never happen.
-		} else if npc >= pcLimit {
+		} else if npc == pcLimit {
```

Comment:
wait is this correct? don't we need a test for this?

Reply:
That's this line in the spec:

* https://github.com/ethereum/consensus-specs/blob/8e56aa760d14ae5e596c265b898eb7b68f500dc4/specs/electra/beacon-chain.md?plain=1#L1651-L1653

Using `>=` is fine but technically it's not what the spec does.

Reply:
`>=` may be safer here. Does `NumPendingConsolidations` ensure linearity?

Reply:
Are you planning to address this? @rkapka

Reply:
this change is no longer in the PR